core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1694,46 +1775,23 @@ public NodeRepresentation addNode(
         }
     }
 
-    private static String formatFrame(WindowNode.Frame frame)
+    private static <T> String formatCollection(Collection<T> collection)
```

Comment:
I think if you make it `formatCollection(Collection<T> collection, Function<T, String> formatter)` it would be harder to use this method without anonymization + you wouldn't need to construct input collection explicitly for anonymization